packages/containerd/containerd-cri-base-json

```diff
@@ -8,12 +8,12 @@
         "cwd": "/",
         {{~#if settings.oci-defaults.capabilities~}}
         "capabilities": {
-            {{~oci_defaults settings.oci-defaults.capabilities~}}
+            {{~oci_defaults settings.oci-defaults.capabilities "containerd"~}}
         },
         {{~/if~}}
         {{~#if settings.oci-defaults.resource-limits~}}
         "rlimits": [
-            {{~oci_defaults settings.oci-defaults.resource-limits~}}
+            {{~oci_defaults settings.oci-defaults.resource-limits "containerd"~}}
```

Comment:
nit: I would probably reverse the argument order, so it's immediately clear what "mode" the helper is called with